docs/customize/vocabularies/users.md

```diff
@@ -31,3 +31,16 @@ The content of the file is as follows:
 If the file is not provided, InvenioRDM creates an admin user with the email
 `admin@inveniosoftware.org` (and a random password). If the file is provided
 but is empty, no default user is created.
+
+To change the randomly generated password, you can created a new shell with `pipenv run invenio shell` and run:
```

Comment:
What about breaking things up here because there is a couple of things intertwined:

1- We don't document how via <https://github.com/inveniosoftware/invenio-rdm-records/blob/019ca5fd28acbafd12e8ba6651b24a85a4db9d4f/invenio_rdm_records/fixtures/users.py#L26>, the `RDM_RECORDS_USER_FIXTURE_PASSWORDS` configuration can be used to define a password other than a randomly generated one. We should document that.
2- A randomly generated password is created for any user without a password in fixtures: admin or not. Making that clearer could help too.
3- I like providing the docs you added for when you just want to change a password. Making the fact that this recipe can be used in any context would make its usefulness really clear to readers.

Reply:
@fenekku I rewrote this section, being more explicit, WDTY? Any suggestion?

![Screenshot 2022-02-21 at 17 26 51](https://user-images.githubusercontent.com/2089455/154994206-149c77c3-063f-41a7-9f27-7207c0a4b5af.png)